master/reference/public-cloud/gce.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Deploying Calico on GCE
+---
+
+To deploy Calico in [Google Compute Engine][GCE], you must ensure that the
+proper firewall rules have been made and that traffic between containers on
+different hosts is not dropped by the GCE fabric. There are a few different
+options for doing this depending on your deployment.
+
+#### Configure GCE Firewall Rules
+
+Calico requires the following firewall rules to function in GCE.
+
+| Description      | Protocol | Port Range |
+|:-----------------|:---------|:-----------|
+| BGP              | TCP      | 179        |
+| \*IPIP           | 4        | all        |
+
+>\*IPIP: This rule is required only when using Calico with IPIP encapsulation.
+Keep reading for information on when IPIP is required in GCE.
+
+#### Routing Traffic
+
+One of the following options must be utilized when using Calico in
+GCE to ensure container traffic is allowed by the GCE network fabric.
+
+##### IP-in-IP encapsulation
+
+Container traffic routing can be enabled by setting [IP-in-IP encapsulation][IPIP]
+and NAT outgoing on the configured Calico IP pools.  Setting these ensures
+that all traffic to and from containers will be sent with IP addresses that
+are allowed by the GCE network fabric.
+
+See the [IP pool configuration reference][IPPool]
+for information on how to configure Calico IP pools.
+
+##### GCE cloud routes
+
+Traffic routing in GCE can be achieved by utilizing GCE cloud routes.
+Kubernetes GCE cloud provider integration simplifies route configuration
+by enabling Kubernetes to handle creating routes.
```

Comment:
I'd just add a but of context saying adding ".. and running Calico in policy-only mode"